.github/workflows/ci.yml

```diff
@@ -0,0 +1,161 @@
+on:
+  push:
+    tags:
+      # Full version
+      - "[0-9]+.[0-9]+.[0-9]+"
+      # Prerelease version
+      - "[0-9]+.[0-9]+.[0-9]+-*"
+
+  pull_request:
+    branches:
+      # Trigger on pull requests into main/master
+      - main
+      - master
+    types: [ opened, synchronize ]
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - 1.57.0
```

Comment:
Does this align with the cargo manifest msrv? If not, that should be one a follow up PR, of which I can take care if

Reply:
It ought to - I'll double check when I'm back at a PC

Reply:
According to `cargo msrv` the actual MSRV is 1.56.1, so we should probably run with that.

Reply:
And also tie it down in the manifest

Reply:
Ignore that, `cargo msrv` is blatantly lying 😂. I'll revert and make it 1.57.0 again.

```
`time v0.3.11` cannot be built because it requires rustc 1.57.0 or newer, while the currently active rustc version is 1.56.1
```